.circleci/config.yml

```diff
@@ -5,6 +5,23 @@ orbs:
   architect: giantswarm/architect@0.8.3
 
 jobs:
+  build:
+    docker:
+      - image: quay.io/giantswarm/architect
+    steps:
+      - checkout
+      - run:
+          name: Install kustomize
+          command: |
+            CGO_ENABLED=0 go install sigs.k8s.io/kustomize/kustomize/v3
```

Comment:
Is there no `kubectl` in here? `kustomize` is included in kubectl version 1.16+ IIRC

Reply:
I didn't see it. Adding kubectl/kustomize to architect would be an option though. I'll confirm.

Reply:
Not a massive deal either way.

Reply:
Keep in mind the kustomize version in kubectl is older than the standalone version

Reply:
I don't see `kubectl` in `architect` and I don't want to bring it in just for this job so I'll stick with this approach if that's alright.